data/5e/wizards_of_the_coast/core/players_handbook/ph_races.lst

```diff
@@ -5,7 +5,7 @@
 Dwarf		SORTKEY:Base_PC	SIZE:M	MOVE:Walk,25			BONUS:STAT|CON|2				ABILITY:Internal|AUTOMATIC|Racial Traits ~ Dwarf	LEGS:2	HANDS:2	RACETYPE:Humanoid	RACESUBTYPE:Dwarf		TYPE:Humanoid.PC	SOURCEPAGE:p.18
 Elf		SORTKEY:Base_PC	SIZE:M	MOVE:Walk,30			BONUS:STAT|DEX|2				ABILITY:Internal|AUTOMATIC|Racial Traits ~ Elf		LEGS:2	HANDS:2	RACETYPE:Humanoid	RACESUBTYPE:Elf		TYPE:Humanoid.PC	SOURCEPAGE:p.21
 Halfling	SORTKEY:Base_PC	SIZE:S	MOVE:Walk,25			BONUS:STAT|DEX|2				ABILITY:Internal|AUTOMATIC|Racial Traits ~ Halfling	LEGS:2	HANDS:2	RACETYPE:Humanoid	RACESUBTYPE:Halfling	TYPE:Humanoid.PC	SOURCEPAGE:p.26
-Human		SORTKEY:Base_PC	SIZE:M	MOVE:Walk,30									ABILITY:Internal|AUTOMATIC|Racial Traits ~ Human	LEGS:2	HANDS:2	RACETYPE:Humanoid	RACESUBTYPE:Human		TYPE:Humanoid.PC	SOURCEPAGE:p.29
+Human		SORTKEY:Base_PC	SIZE:M	MOVE:Walk,30			BONUS:STAT|STR,DEX,CON,INT,WIS,CHA|1						ABILITY:Internal|AUTOMATIC|Racial Traits ~ Human	LEGS:2	HANDS:2	RACETYPE:Humanoid	RACESUBTYPE:Human		TYPE:Humanoid.PC	SOURCEPAGE:p.29
```

Comment:
This is best split into six tags for the conversion process. I will accept as-is, but know there is a logic to splitting it up.